src/LibraryViewExtension/web/library/library.html

```diff
@@ -39,7 +39,6 @@
                 let layoutSpecsJson = downloaded["layoutSpecs"];
 
                 if (!loadedTypesJson || (!layoutSpecsJson)) {
-                    console.log("Error: Couldn't get enough data to load library.");
```

Comment:
removing this because it's not an error, this logic is expected in the librarie.js repo